### ISSUE TITLE
NTR: Swelling, highly-localized on limb (not exclusively joint) (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7971]
New term request Swelling, highly-localized on limb (not exclusively joint) (ang_130715135134): Extremity swollen or mass, single; arm; leg; not as generalized as edema; bruise; bruising, soft tissue  UMLS: 1411776

**Comment by pnrobinson** [comment]
@MickeySegal
Not sure I follow the definition. What is the disease context?

**Comment by MickeySegal** [comment]
We use this for a large number of conditions that cause localized swelling, including fibrosarcoma, Ewing sarcoma, calcaneus fractures, fibrodysplasia ossificans progressiva and biceps tendon rupture.

**Comment by pnrobinson** [comment]
Would something like this be good

Localized soft-tissue swelling on extremity
def: Abnormal enlargement (swelling) that is restricted to a specific part of a limb.
Comment: This term is meant to describe new-onset swelling. For instance, patients with Ewing sarcoma often present with local symptoms such as pain, stiffness, or swelling for a few weeks or months.

parent: [Abnormality of limbs HP:0040064](https://hpo.jax.org/app/browse/term/HP:0040064)


 ain and/or swelling, usually in the arms, legs, chest, back, or pelvis.

**Comment by MickeySegal** [comment]
That works.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000840
- label: Localized soft-tissue swelling on extremity
- parents: [Abnormality of limbs (HP:0040064)](https://hpo.jax.org/app/browse/term/HP:0040064)
- pmid: 
- def: Abnormal enlargement (swelling) that is restricted to a specific part of a limb.
- comment: This term is meant to describe new-onset swelling. For instance, patients with Ewing sarcoma often present with local symptoms such as pain, stiffness, or swelling for a few weeks or months.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7971
- synonyms:
